nasa/harmony
# Harmony 1508 item update service

## Jira Issue ID
HARMONY-1508

## Description
Moves the work-item updater tasks into k8s.

## Local Test Steps
1. Delete your `harmony` k8s cluster using `bin/stop-harmony-and-services`
2. Run `bin/bootstrap-harmony` 
3. verify that the large and small work-item updater services are running using `kubectl get pods`
4. Execute some test queries
5. Use `kubectl logs <pod-id>` to look at the logs of the new service pods and verify that they are processing the queues
and handling work-item updates
6. Verify that your requests complete

7. (Optional but someone should do it) use the `harmony-1508` branch of `harmony-ci-cd` with this branch to do a sandbox deployment and repeat the tests above

## PR Acceptance Checklist
